group_test.go

```diff
@@ -109,3 +109,45 @@ func TestGroupService_Remove(t *testing.T) {
 		t.Errorf("Error given: %s", err)
 	}
 }
+
+func TestGroupService_Create(t *testing.T) {
+	setup()
+	defer teardown()
+	testMux.HandleFunc("/rest/api/2/group", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "POST")
+		testRequestURL(t, r, "/rest/api/2/group")
+
+		w.WriteHeader(http.StatusCreated)
+		fmt.Fprint(w, `{"id": "https://docs.atlassian.com/jira/REST/schema/add-group#","title": "Add Group","type": "object","name" : "hello" ,"additionalProperties": false}`)
```

Comment:
I can't find any documentation like this.  Where is it?  The only schema I see is:
```json
{
  "properties": {
    "name": {
      "type": "string"
    },
    "self": {
      "format": "uri",
      "type": "string"
    },
    "users": {
      "$ref": "#/definitions/PagedListWrapperUserJsonBeanApplicationUser"
    }
  },
  "type": "object"
}
```